esp/scm/ws_esdlconfig.ecm

```diff
@@ -144,7 +146,7 @@ ESPresponse [exceptions_inline] GetESDLBindingResponse
     string EspProcName;   //Name of ESP Process
     string BindingName;
     string EspPort;
-    [depr_var("1.1")] String ConfigXML;
+    String ConfigXML;
```

Comment:
I should have never deprecated... that change was never released so we'll always return ConfigXML